controllers/storagecluster/cephcluster.go

```diff
@@ -273,6 +289,12 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		}
 	}
 
+	// Create the prometheus rules if required by the cephcluster CR
+	if err := createPrometheusRules(r, sc, cephCluster); err != nil {
```

Comment:
I looked at adding a new reconciler for the prometheus rules, but it seemed best to instead make it part of the cephcluster reconciler because it depends on the cephcluster CR settings.

Reply:
Agree, this is better than having a separate reconciler.